pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -233,6 +233,7 @@ def _default_test_id(self):
         if self.root == 'lms':
             default_test_globs.append("{system}/tests.py".format(system=self.root))
             default_test_globs.append("openedx/core/djangolib/*")
+            default_test_globs.append("openedx/core/tests/*")
```

Comment:
Suggestion: I'd move this change to a separate PR and merge it first. It'd be great to have jenkins run this test separately from this PR, just in case we need to revert again.

Reply:
Another suggestion: can we enhance the existing test so it exercises more than one user's access to the admin page, say both superuser and staff user?